implants/Cargo.toml

```diff
@@ -55,12 +58,14 @@ sys-info = "0.9.1"
 sysinfo = "0.29.7"
 tar = "0.4.38"
 tavern = { path = "./lib/tavern" }
+c2 = { path = "./lib/c2" }
 tempfile = "3.3.0"
 tera = "1.17.1"
 thiserror = "1.0.30"
 tokio = "1.19.1"
 tokio-stream = "0.1.9"
 tokio-test = "*"
+tonic = { git = "https://github.com/hyperium/tonic.git", rev = "07e4ee1" }
```

Comment:
Do we have to pin to a commit or is there a version number we can use?

Reply:
This was for testing, when we were attempting to patch the route path bug. Still waiting on a fix for it.